patterns/1-initial/cross-team-retrospectives.md

```diff
@@ -0,0 +1,75 @@
+
+## Patlet
+
+A host team working with contributors outside of their own line of management constantly runs into misunderstandings.
+As a result collaboration becomes brittle and frustrating.
+Setting aside time for regular retrospectives for the InnerSource team consisting of trusted committers and contributors can help make communication smooth.
+
+## Problem
+
+For long running collaborations friction between host team and collaborators is substantially reducing focus and energy for everyone involved.
+Willingness to continue the collaboration is shrinking.
+
+## Context
+
+A host team of trusted committers has started a long running collaboration with a group of contributors.
+
+* Over time the number of misunderstandings grows.
+* People may run into mis-communication.
+* Teams may discover slight differences in development culture.
+* Team members may discover that assumptions they made about the other team are false.
+* Contribution processes may not be entirely clear and workable for everyone involved.
+
+## Forces
+
+* Participants are inclined to take "written over verbal" as "only written".
+* Trusted committers are all part of the same team.
+* There is a group of contributors all coming from the same team.
+* As a result trusted committers know each other well and understand constraints, prioritization side effects and team dynamics without ever sharing them with contributors.
+* Also contributors form a well knit group.
+* The contribution process is seen as transient and temporary.
+As a result little is invested in forming a shared team of trusted committers and contributors.
+* There is no clear path from contributor to becoming trusted committer - other than becoming a member of the host team.
+
+## Solution
+
+Bring host team and contributors together:
+
+* As a first step it can help to share a meal together and get to know each other.
+* For collaborations running over several weeks establish a monthly 30 minute retrospective meeting that involves everyone who is needed for a successful contribution.
+* Make sure that action items for each restrospective are being followed up upon, ideally check these action items at the beginning of the next retrospective.
+* Keep the agenda of retrospectives stable and predictable: It's already uncomfortable enough to name and resolve collaboration issues.
+
+Example agenda:
+
+* 5 minute checkin so everyone can test their audio setup, silly questions preferred so people can laugh together, reducing overall stress.
+* 5 minute review for action items from last meeting (each item presented by its owner)
+* 10 minutes to gather strengths and weaknesses of the past collaboration time period. Do this as a combination of writing (sticky notes on a digital white board) and verbally explaining the stickies to make sure introverts get involved as well.
+* 2 minutes to put dots against weaknesses that should be addressed in the next cycle.
+Pick the top 1-2 weaknesses.
```

Comment:
```suggestion
* 2 minutes to put dots against weaknesses that should be addressed in the next cycle. Pick the top 1-2 weaknesses.
```

gitbook won't format this correctly unfortunately. So got to put sentences within a bullet onto the same line.